README.md

```diff
@@ -365,7 +511,7 @@ inside Amazon VPC.](https://docs.aws.amazon.com/kinesisanalytics/latest/java/vpc
   <dependency>
        <groupId>software.amazon.glue</groupId>
        <artifactId>schema-registry-flink-serde</artifactId>
-       <version>1.0.2</version>
+       <version>1.0.1</version>
```

Comment:
Is this correct?

Reply:
This isn't, we are not upgrading flink package dependency because we are not maintaining flink from this repo anymore. Flink Connectors belong to Apache Flink repo for GSR.